client/client.go

```diff
@@ -287,9 +287,18 @@ func Configure(logger hclog.Logger, providerConfig interface{}) (schema.ClientMe
 	awsConfig := providerConfig.(*Config)
 	client := NewAwsClient(logger, awsConfig.Accounts, awsConfig.Regions)
 
-	if len(client.regions) == 0 {
+	// validate regions values
+	for i, region := range client.regions {
+		if i != 0 && region == "*" {
+			return nil, fmt.Errorf("region wildcard \"*\" is only supported in 0 index")
```

Comment:
```suggestion
			return nil, fmt.Errorf("region wildcard \"*\" is only supported as first argument")
```
(Let's make it user friendly)